src/default_panels/StyleColorbarsPanel.js

```diff
@@ -6,17 +6,17 @@ import {Radio, TextEditor, TraceAccordion, Fold, Panel} from '../components';
 import {localize} from '../lib';
 
 const StyleTracesPanel = ({localize: _}) => (
-  <TraceAccordion>
-    <Panel>
-      <Radio
-        attr="showscale"
-        options={[
-          {label: _('Show'), value: true},
-          {label: _('Hide'), value: false},
-        ]}
-      />
+  <TraceAccordion messageIfEmptyFold="Need a color for a colorbar!">
+    <Radio
```

Comment:
but shouldn't this message be more individualized, what if it's not a color bar fold that is empty..

Reply:
This is in the ColorBar panel only...

Reply:
and it's needed in TraceAccordion? I thought TraceAccordion was just had a colorscale widget for when there's marker.colorscale..? there's a whole fold for color bars in traceAccordion?

Reply:
The TraceAccordion generates the panel and the folds so if we want the individual colorbar folds to have a message-if-empty this is where to pass it in. But look at the filename: this is a highly-inidividualized case :)

Reply:
oh sorry, didn't see the filename `StyleColorbarsPanel.js` 👍